src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/pipeline_management/process_deployment_map.py

```diff
@@ -74,6 +74,7 @@ def start_executions(sfn_client, deployment_map):
     )
     for pipeline in deployment_map.get("pipelines"):
         LOGGER.debug("Payload: %s", pipeline)
+        pipeline = {**pipeline, "deployment_map_source": "s3"}
```

Comment:
`s3` is lower case here, while in the other hard coded paths they use `S3` instead.